crawler/processing/create_plate_validator.py

```diff
@@ -0,0 +1,213 @@
+import logging
```

Comment:
Nice. Putting all of the validation in a single class makes it easier to understand and hopefully makes it more maintainable.

Reply:
Another benefit is that the errors associated with the validation can be kept as a class variable.  I was originally storing them on the message, but the message is supposed to be generic to the contents until you know how to decode it which made doing things like counting samples more complicated.